pkg/registry/apis/provisioning/register.go

```diff
@@ -117,6 +122,10 @@ func RegisterAPIService(
 		return nil, nil // skip registration unless opting into experimental apis OR the feature specifically
 	}
 
+	if !features.IsEnabledGlobally(featuremgmt.FlagUnifiedStorageSearch) {
+		return nil, fmt.Errorf("missing featureFlag: unifiedStorageSearch")
```

Comment:
This requires:
```

[feature_toggles]
...
unifiedStorageSearch = true
...
```
I'm not sure if we should fail here... but at least for now want to be as loud as possible since things won't work without it

Reply:
shouldn't this be enabled by us if the enable provisioning? I thought we wanted to keep a single flag also with our requirements.

Reply:
Eventually yes -- but I think for now making `unifiedStorageSearch` a requirement is OK.  It is on the cusp of getting stable enough to consider enabling by default; and will need to be on by default before G12

Reply:
ok, accumulating flags and deps till then 😂